Makefile

```diff
@@ -88,6 +89,9 @@ requirements:
 	if [ "$(pip list | grep pipreqs | wc -l)" != "0" ]; then \
 		rm -f src/requirements.txt; \
 		pipreqs ./; \
+		echo "coverage==5.1" >> requirements.txt; \
+		echo "oyaml==0.9" >> requirements.txt; \
+		echo "flake8==3.7.9" >> requirements.txt; \
```

Comment:
I think I mentioned this somewhere else as well, but we should consider maintaining a separate test-requirements.txt with these. Then .... (see below)